### ISSUE TITLE
Request to provide a setting to stop auto save feature

### ISSUE THREAD

**Issue by kelvin-hk** [RobertoMachorro/Moped#32]
I love your editor, but the Auto save feature make me in trouble. It is easy to make careless mistake.
Can you provide a setting in Preferences to stop the Auto Save? Manual save is good enough for me. 

Thank you.

**Comment by RobertoMachorro** [comment]
Good point, I'm compiling a list of features that will next be implemented. This one will be.

**Comment by RobertoMachorro** [comment]
Flagging this as a bug, the editor auto-saves but it doesn't reflect so in the document.
